Utils/hdinsight-node-common/Test/resources/com/microsoft/azure/hdinsight/spark/common/CosmosServerlessSparkBatchJobScenario.feature

```diff
@@ -5,4 +5,9 @@ Feature: Cosmos Serverless Spark Batch Job Testing
     And setup a mock cosmos serverless service for 'GET' request '/activityTypes/spark/batchJobs/46c07889-3590-48f8-b2bc-7f52622b5a0b?api-version=2018-02-01-preview' to return '{"properties":{"responsePayload":{"id":10,"appInfo":{"driverLogUrl":null,"sparkUiUrl":null},"log":[],"state":"started"},"sparkMasterUI":"https://46c07889-3590-48f8-b2bc-7f52622b5a0b.azuredatalakeanalytics.net"},"id":"46c07889-3590-48f8-b2bc-7f52622b5a0b","priority":1000,"name":"Session_46c07889-3590-48f8-b2bc-7f52622b5a0b","activityType":"spark","submitter":"rufan","analyticsUnits":9,"submitTime":"2018-11-12T23:57:38.327Z","startTime":"2018-11-12T23:57:38.827Z","endTime":"2018-11-13T00:03:20.183Z","state":"Ended"}' with status code 200
     And submit a cosmos serverless spark batch job
     Then verify 'PUT' request for '/activityTypes/spark/batchJobs/46c07889-3590-48f8-b2bc-7f52622b5a0b?api-version=2018-02-01-preview'
-    Then batch ID should be 10
```

Comment:
This `THEN` is removed?

Reply:
Add comment: `For HDInsight job , we can get batch ID immediatelly after we submit job, but for Serverless job,  some time are needed for environment setup before batch ID is available`